microsoft-edge/webview2/samples/webview2_sample_uwp.md

```diff
@@ -233,4 +233,4 @@ At the top of Visual Studio, set the build target, as follows:
 <!-- ====================================================================== -->
 ## See also
 
```

Comment:
This article might need a major update to remove "prerelease".  I'll confirm & update it.